addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxHandler.java

```
@@ -118,9 +126,14 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     break;
                 case REBOOT:
                     setReboot(command);
+                default:
+                    logger.debug("Thing {}: unexpected command {} from channel {}", getThing().getUID(), command,
+                            channelUID.getId());
+                    break;
             }
-        } catch (FreeboxException e) {
-            logger.error("{}", e.getMessage());
+        } catch (Exception e) {
```

My intention was to catch all exceptions to display an appropriate log message.